packages/loot-core/package.json

```diff
@@ -50,6 +50,7 @@
     "@types/jest": "^27.5.0",
     "@types/jlongster__sql.js": "npm:@types/sql.js@latest",
     "@types/pegjs": "^0.10.3",
+    "@types/uuid": "^9.0.2",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏we are actually using uuid v3 in Actual. Can we either upgrade it to v9 or downgrade the types to v3? Both of these things should +- match.

Reply:
Updated to v9!